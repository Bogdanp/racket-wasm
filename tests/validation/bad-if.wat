(module
 (export "main" (func $main))
 (func $main
   (result i32)
   (i32.const 1)
   (if
    (then
     (i32.const 42)
     (drop)))))
