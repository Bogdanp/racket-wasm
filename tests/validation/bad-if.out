(#f . "result arity error\n  expected: [i32]\n  found: []\n  at: (funcidx 0)")